extensions/provisioning/ssh.go

```diff
@@ -81,6 +102,79 @@ func CallSSHTestByName(testCase provisioninginput.SSHTestCase, node *nodes.Node,
 		}
 
 		return err
+
+	//This test checks if the audit log file is properly created on the node (skipped if its not a control plane node).
+	case auditLog:
+		if node.NodeLabels[controlPlaneLabel] != "true" {
+			logrus.Infof("Node %s is not a control-plane node, skipping", node.PublicIPAddress)
+			return nil
+		}
+
+		logrus.Infof("Running audit log test on node %s", node.PublicIPAddress)
+		cluster, err := client.Steve.SteveType(clusterManagementSteveType).ByID(clusterID)
+		if err != nil {
+			return err
+		}
+
+		auditLogPath := rancherDir + cluster.Labels[providerLabel]
+		auditLogFile := ""
+		if cluster.Labels[providerLabel] == "rke2" {
```

Comment:
Can you put `rke2` in a `const` block?

Reply:
Yes, once my other PR gets merged I plan to rebase and expand those constant files see [defaults package](https://github.com/rancher/shepherd/pull/75/files#diff-19cc5dc9b440ac24f6f180266d376ca3e83b334f7fbef726bf7f4d0dbfe371e7) in this PR